src/Geom_Poly.f90

```diff
@@ -156,6 +158,13 @@ MODULE Geom_Poly
   !> @copydetails Geom_Poly::isequal_PolygonType
   MODULE PROCEDURE isequal_PolygonType
 ENDINTERFACE
+
+!> @brief
```

Comment:
Also should fill this in if you're going to have it

Reply:
Great, thanks for filling all that in.  There were a lot more empty ones than I realized.